### ISSUE TITLE
[Android crash] com.mapbox.mapboxsdk.maps.NativeMapView.createSurface

### ISSUE THREAD

**Issue by henrikra** [nitaliano/react-native-mapbox-gl#903]
We have been getting this crash lately. 9 times totally.

```
java.lang.NullPointerException Attempt to invoke virtual method 'void com.mapbox.mapboxsdk.maps.NativeMapView.createSurface(android.view.Surface)' on a null object reference 
    MapView.java:427 com.mapbox.mapboxsdk.maps.MapView$SurfaceCallback.surfaceCreated
    SurfaceView.java:597 android.view.SurfaceView.updateWindow
    SurfaceView.java:243 android.view.SurfaceView.onWindowVisibilityChanged
    View.java:9653 android.view.View.dispatchWindowVisibilityChanged
    ViewGroup.java:1310 android.view.ViewGroup.dispatchWindowVisibilityChanged
    ViewGroup.java:1310 android.view.ViewGroup.dispatchWindowVisibilityChanged
    ViewGroup.java:1310 android.view.ViewGroup.dispatchWindowVisibilityChanged
    ViewGroup.java:1310 android.view.ViewGroup.dispatchWindowVisibilityChanged
    ViewGroup.java:1310 android.view.ViewGroup.dispatchWindowVisibilityChanged
    ViewGroup.java:1310 android.view.ViewGroup.dispatchWindowVisibilityChanged
    ViewGroup.java:1310 android.view.ViewGroup.dispatchWindowVisibilityChanged
    ViewGroup.java:1310 android.view.ViewGroup.dispatchWindowVisibilityChanged
    ViewGroup.java:1310 android.view.ViewGroup.dispatchWindowVisibilityChanged
    ViewGroup.java:1310 android.view.ViewGroup.dispatchWindowVisibilityChanged
    ViewGroup.java:1310 android.view.ViewGroup.dispatchWindowVisibilityChanged
    ViewGroup.java:1310 android.view.ViewGroup.dispatchWindowVisibilityChanged
    ViewRootImpl.java:1410 android.view.ViewRootImpl.performTraversals
    ViewRootImpl.java:1134 android.view.ViewRootImpl.doTraversal
    ViewRootImpl.java:6050 android.view.ViewRootImpl$TraversalRunnable.run
    Choreographer.java:860 android.view.Choreographer$CallbackRecord.run
    Choreographer.java:672 android.view.Choreographer.doCallbacks
    Choreographer.java:608 android.view.Choreographer.doFrame
    Choreographer.java:846 android.view.Choreographer$FrameDisplayEventReceiver.run
    Handler.java:742 android.os.Handler.handleCallback
    Handler.java:95 android.os.Handler.dispatchMessage
    Looper.java:154 android.os.Looper.loop
    ActivityThread.java:5529 android.app.ActivityThread.main
    Method.java:-2 java.lang.reflect.Method.invoke
    ZygoteInit.java:739 com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run
    ZygoteInit.java:629 com.android.internal.os.ZygoteInit.main
```

**Comment by henrikra** [comment]
@nitaliano Do you have any idea for this? We are still getting this on production :/

**Comment by nitaliano** [comment]
this should be fixed with `6.1.0` the version of the Android SDK that is used has resolved this issue. Going to close out reopen if you're still seeing it
